<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1502</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>850</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="line_serach">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>1331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QTableWidget" name="covid_table">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>1431</width>
       <height>291</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>날짜</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>국가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>신규 확진자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>누적 확진자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>신규 사망자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>누적 사망자</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>40</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>1310</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_del">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_change">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>수정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_graph">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>그래프</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>514</x>
       <y>460</y>
       <width>481</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="graph_verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>1004</x>
       <y>460</y>
       <width>481</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="graph_verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>460</y>
       <width>481</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="graph_verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="date">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>날짜</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="country">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>국가</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_cases">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>신규 확진자</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cumulative_cases">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>360</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>370</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>누적 확진자</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_deaths">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>450</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cumulative_deaths">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>550</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>신규 사망자</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>560</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>누적 사망자</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_complete">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>660</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>완료</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_cancle">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>660</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_serach</tabstop>
  <tabstop>btn_search</tabstop>
  <tabstop>covid_table</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_del</tabstop>
  <tabstop>date</tabstop>
  <tabstop>country</tabstop>
  <tabstop>new_cases</tabstop>
  <tabstop>cumulative_cases</tabstop>
  <tabstop>new_deaths</tabstop>
  <tabstop>cumulative_deaths</tabstop>
  <tabstop>btn_complete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
